source/wp-content/themes/wporg-news-2021/sass/blocks/_post-author.scss

```diff
@@ -1,3 +0,0 @@
-.wp-block-post-author__name {
-	font-weight: var(--wp--custom--post-author--font-weight);
```

Comment:
This value can also be removed from `theme.json`, since it's not used anywhere else.

Reply:
Good catch!

Reply:
https://github.com/WordPress/wporg-news-2021/pull/328/commits/40ac25689a864d657212e0c9de56575de02ea701